experimental/examples/imageviewer/androidApp/src/androidMain/AndroidManifest.xml

```diff
@@ -12,11 +12,11 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_imageviewer_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.AppCompat.Light.NoActionBar">
-        <activity
-            android:exported="true"
-            android:name="example.imageviewer.MainActivity"
-        >
+        android:theme="@style/Theme.ImageViewer">
+
+        <meta-data android:name="com.google.android.geo.API_KEY" android:value="${MAPS_API_KEY}" />
```

Comment:
Please add a comment with instruction what to do with `MAPS_API_KEY`

Reply:
Already added in README

Reply:
What happens when you don't provide a MAPS_API_KEY? Will it just gracefully show an empty map? (Let's just make sure it doesn't crash)

Reply:
- Get a [Google Maps API key](https://developers.google.com/maps/documentation/android-sdk/get-api-key)
 - Add to local.properties: `sdk.dir=YOUR_SDK_PATH`
 - Create a file in the root directory named `local.properties` with a single line that looks like
   this, replacing YOUR_KEY with the key from step 1: `MAPS_API_KEY=YOUR_KEY`
 - Open project in IntelliJ IDEA or Android Studio and run "android" configuration.

Reply:
@SebastianAigner 
Yes, it will show hust emty map

Reply:
Yeah, but as an user I'd expect a comment like "Change this value in local.properties file" here